tests/main/snap-quota-journal/test-snapd-journal-quota/bin/logger

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+i=0
+while true; do
+    echo "LogDaemon: message number: $i"
+    i=$((i+1))
+    sleep 1
```

Comment:
```suggestion
    sleep 0.1
```

Reply:
I moved to 0.2 seconds as a compromise :)

Reply:
Fun times - turns out making this faster will hit the journal limit and that will mean the echo gets a SIGPIPE and the service dies. So I reverted back to 1s